runner-from-userdata.tf

```diff
@@ -59,8 +59,8 @@ data "template_file" "gitlab_runner" {
     runners_use_private_address = "${var.runners_use_private_address}"
     bucket_name                 = "${aws_s3_bucket.build_cache.bucket}"
     runner_environment_tag      = "${var.environment}"
-
-    runners_config = "${data.template_file.runners.rendered}"
+    instance-profile-name       = "${aws_iam_instance_profile.instance.name}"
```

Comment:
we can have a `resource "aws_iam_role" "instance"` name as output and attach any custom policies to in at service level. this role automatically assigned to all spawned machines